### PR TITLE
Add an example of how to use kind

### PR DESCRIPTION
**Pull request by Gsantomaggio**
## Proposed Changes
Add an example of how to use [`kind`](https://github.com/kubernetes-sigs/kind)
Kind is K8s tools  for local tests and CI integrations

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes issue #NNNN)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause an observable behavior change in existing systems)
- [x] Documentation improvements (corrections, new content, etc)
- [ ] Cosmetic change (whitespace, formatting, etc)

## Checklist

- [x] I have read the `CONTRIBUTING.md` document
- [x] I have signed the CA (see https://cla.pivotal.io/sign/rabbitmq)
- [ ] All tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in related repositories

## Further Comments

I wanted to push this example on my personal repository, but I thought that can be useful for the community. As always, feel free to close/ignore it :) 
Thank you

### CODE REVIEW

**Review by yashbhutwala — APPROVED** [review]
Nice change, especially like the persistence addition 😃

File: examples/kind/README.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+The example uses, targets or assumes:
+
+ * [Kind](https://github.com/kubernetes-sigs/kind) 
+ * [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 
+ 
+## Quick Start
+
+ * Create Kind cluster 
+ ```
+  $ kind create cluster --config kind/kind-cluster/kind-cluster.yaml
```
Comment (michaelklishin) [inline examples/kind/README.md]: We don't use `$` to indicate a shell input in our examples. Please drop it.

File: examples/kind/README.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+```
+
+Deploy RabbitMQ with or without [persistent-volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
```
Comment (michaelklishin) [inline examples/kind/README.md]: The hyphen in `persistent-volumes` seems unnecessary?

File: examples/kind/README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+The `NodePort` service exposes the ports: 
+```yaml
+spec:
+  type: NodePort
+  ports:
+   - name: http
+     protocol: TCP
+     port: 15672
+     targetPort: 15672
+     nodePort: 31672 <---- binds the extraPortMappings.containerPort 31672
```
Comment (michaelklishin) [inline examples/kind/README.md]: Can this be a regular YAML comment?

File: examples/kind/README.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+$ kubectl get pv -n rabbitmq-dev-persistence
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                                               STORAGECLASS   REASON   AGE
+pvc-0db837ec-a856-4a6b-9acf-2c9110bb9f12   1Gi        RWO            Delete           Bound    rabbitmq-dev-persistence/rabbitmq-data-rabbitmq-0   standard                2m41s
+pvc-24514f31-5b40-4bd6-a721-84a16afaa697   1Gi        RWO            Delete           Bound    rabbitmq-dev-persistence/rabbitmq-data-rabbitmq-1   standard                81s
+pvc-2b7162f5-c596-404d-be85-475600b9f82a   1Gi        RWO            Delete           Bound    rabbitmq-dev-persistence/rabbitmq-data-rabbitmq-2   standard                5s
+```
+
+## CleanUp
+
+CleanUP RabbitMQ without PV:
```
Comment (michaelklishin) [inline examples/kind/README.md]: It should either be "clean-up" (a noun) or "clean up" (a verb). The title is probably meant to be a noun. CleanUP sounds like an unfortunate product name 🤣

**Review by michaelklishin — CHANGES_REQUESTED** [review]
Thank you. This could use a bit of editing. I'm happy to do some rewording after merging but let's address some things that immediately stand out first.

**Comment by Gsantomaggio** [comment]
ok done!

**Review by michaelklishin — APPROVED** [review]

**Comment by michaelklishin** [comment]
Thank you!

**Pull request merged**